sweep/store.go

```diff
@@ -0,0 +1,49 @@
+package sweep
+
+import (
+	"github.com/btcsuite/btcd/chaincfg/chainhash"
+	"github.com/btcsuite/btcd/wire"
+	"github.com/lightningnetwork/lnd/channeldb"
+)
+
+// SweeperStore stores published txes.
+type SweeperStore interface {
+	IsOurTx(hash chainhash.Hash) bool
+
+	NotifyPublishTx(*wire.MsgTx) error
+
+	GetLastPublishedTx() (*wire.MsgTx, error)
```

Comment:
Will we only have a single outstanding transaction that hasn't yet been confirmed?

Reply:
Gotcha, understood now.